src/shared/constants/nodeData.ts

```diff
@@ -0,0 +1,63 @@
+export interface NodeData {
+  url: string;
+  provider: string;
+}
+
+export interface AllNodeData {
+  [key: string]: NodeData;
+}
+
+export const allNodeData: AllNodeData = {
+  prime: {
+    url: 'wss://rpc.prime.colosseum.quaiscan.io',
+    provider: 'https://rpc.prime.colosseum.quaiscan.io',
+  },
+  'region-0': {
+    url: 'wss://rpc.cyprus.colosseum.quaiscan.io',
+    provider: 'https://rpc.cyprus.colosseum.quaiscan.io',
+  },
+  'region-1': {
+    url: 'wss://rpc.paxos.colosseum.quaiscan.io',
+    provider: 'https://rpc.paxos.colosseum.quaiscan.io',
+  },
+  'region-2': {
+    url: 'wss://rpc.hydra.colosseum.quaiscan.io',
+    provider: 'https://rpc.hydra.colosseum.quaiscan.io',
+  },
+  'wallet-zone-0-0': {
+    url: 'wss://rpc.cyprus1.colosseum.quaiscan.io',
+    provider: 'https://rpc.cyprus1.colosseum.quaiscan.io',
+  },
+  'wallet-zone-0-1': {
+    url: 'wss://rpc.cyprus2.colosseum.quaiscan.io',
+    provider: 'https://rpc.cyprus2.colosseum.quaiscan.io',
+  },
+  'wallet-zone-0-2': {
+    url: 'wss://rpc.cyprus3.colosseum.quaiscan.',
+    provider: 'https://rpc.cyprus2.colosseum.quaiscan.io',
```

Comment:
question: shouldn't this be with domain `cyprus3`?

```suggestion
    provider: 'https://rpc.cyprus3.colosseum.quaiscan.io',
```

Reply:
good catch!